.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Suggested this part gets moved to a `.babelrc` as mentioned here: https://babeljs.io/docs/en/config-files